packages/beacon-node/test/e2e/network/reqresp.test.ts

```diff
@@ -33,24 +33,21 @@ function runTests({useWorker}: {useWorker: boolean}): void {
     ...chainConfig,
     ALTAIR_FORK_EPOCH: 0,
   });
+  let controller: AbortController;
 
   const afterEachCallbacks: (() => Promise<void> | void)[] = [];
+
+  beforeEach(() => {
+    controller = new AbortController();
+  });
```

Comment:
do it make sense to abort the controller in `afterEach`?

Reply:
Had it before but does not make sense of it with these changes. Mostly it's used for sleep and peer connection, if any of those hangs up the test will already timeout.